databricks_cli/jobs/cli.py

```diff
@@ -87,10 +87,14 @@ def reset_cli(api_client, json_file, json, job_id, version):
         with open(json_file, 'r') as f:
             json = f.read()
     deser_json = json_loads(json)
-    request_body = {
-        'job_id': job_id,
-        'new_settings': deser_json
-    }
+    if 'new_settings' in deser_json:
+        request_body = deser_json
+        request_body['job_id'] = job_id
```

Comment:
What about checking for `new_clusters` as the title implies? If you specify a dict with only that key, it would be wrapped in the `new_settings` field in the else branch. I suspect you may need to check for more than just the `new_settings` key here to determine what is actually specified.